packages/pwa/src/components/styleguide/Colors.stories.tsx

```
@@ -0,0 +1,159 @@
+import React from 'react';
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(@typescript-eslint/consistent-type-imports)